code/cynthia/Python/test_ATM_lab12.py

```diff
@@ -0,0 +1,37 @@
+#Automated Tests
+#using using ATM 
+
+from lab08_2 import ATM
+import pytest
+from unittest.mock import patch 
+
+def test_ATM_creation():
+    atm = ATM(0)
+    assert atm.balance == 0
+    atm = ATM(-5)
+    assert atm.balance == -5
+    atm = ATM(.1)
+    assert atm.interest_rate == .1
```

Comment:
I feel like this assertion implies that the interest rate is set by you calling it with `.1` as the argument, but it's actually because all instances of atm have that interest rate. The `__init__` only takes one positional argument, for balance. I would solve this by just getting rid of line 13.